docs/summary_table_of_scores.md

```diff
@@ -0,0 +1,3 @@
+| continuous | probability | statistical tests      |
+| ---------- | ----------- | ----------- |
+| MAE, MSE, RMSE, FIRM | CRPS, FIRM |  Diebold Mariano (with the Harvey 1997 and Hering 2011 modifications) |
```

Comment:
"Hering and Genton"

Reply:
Again, FIRM is primarily categorical. Can you put this table in a common place so that we don't have to update it in two places.

Reply:
I have updated this to read 

Diebold Mariano (with the Harvey et al. 1997 and the Hering and Genton 2011 modifications)

And updated the table in both places. Unfortunately for technical reasons I can't put them in a single place right now.

The table has been updated to show FIRM as categorical.